src/SkyApm.Core/SkyLogging/SkyApmLogger.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Microsoft.Extensions.Logging;
```

Comment:
The `SkyApm.Core` project should not depend on `Microsoft.Extensions.Logging`

Reply:
A better approach is to implement the Skywalking-log adapter in a new solution project